atd-vze/src/views/Crashes/RelatedRecordsTable.js

```diff
@@ -85,7 +85,9 @@ const RelatedRecordsTable = ({
     const isFieldNull = fieldValue === null;
     const isFieldBlank = fieldValue === "";
 
-    return isFieldNull || isFieldBlank ? "NO DATA" : fieldValue;
+    return isFieldNull || isFieldBlank
+      ? "NO DATA"
+      : fieldValue.toString().toUpperCase();
```

Comment:
i noticed that there were a few crash records that had lowercase text for some of the fields (saying "austin" instead of "AUSTIN") in the related records table which goes against the pattern and looked weird, so i added this to make sure all of the text is rendered in the UI in a consistent way.